INSTALL.rst

```diff
@@ -234,7 +234,7 @@ png2gas
 
 ADIOS
 """""
-- 1.10.0+ (requires *MPI*, *zlib* and `mxml <http://www.msweet.org/projects.php?Z3>`_)
+- 1.10.0+ (requires *MPI* and *zlib*)
```

Comment:
note: mxml is now shipped inside ADIOS itself and does not need to be pre-installed